### ISSUE TITLE
Accessibility: Can't tab to user pulldown menu when logged in

### ISSUE THREAD

**Issue by briangrossman** [mitodl/mitxonline#225]
### Steps to Reproduce

- Log in to MITx 
- Start tabbing through elements on the page

### Expected Behavior

- You should be able to tab to the pulldown and select items with the keyboard only

### Actual Behavior

- You cannot tab to the pulldown from the user's name in the upper right: 
![Screen Shot 2021-09-24 at 3 32 01 PM](https://user-images.githubusercontent.com/21342568/134730587-5a3daa38-55b0-4b54-8ac7-0f789b4d1e7c.png)
- Note that you are able to tab to the hamburger navigation when the browser is narrow enough: 
![Screen Shot 2021-09-24 at 3 31 52 PM](https://user-images.githubusercontent.com/21342568/134730680-b6de23f2-2ba0-4ddc-8c14-795ae4f47a66.png)
- Note you are also able to tab to `Sign In` and `Create Account` when you are not logged in

**Comment by briangrossman** [comment]
Some potentially helpful tips from the folks who found the issue:

----------

The reason the profile button (labeled by your name) is not accessible is because:

- it is not in tab order;
- it has no explicit keyboard handler or isn't an element with intrinsic keyboard access like a button or link;

The easiest fix would be to make it a button element or anchor "a" element. If anchor, then also must have either href="#" or tabindex="0".

`<!-- make this "button" or "a" instead of "div" --> <div class="col-2 dropdown-toggle" id="dropdownMenuButtonDesktop" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Richard Caloggero</div>`

The other fix (assuming they couldn't change the element) would be:

- add tabindex="0";
- add keyboard handler;

The easiest thing to do if adding explicit keyboard handler is to simply have the handler fire a click event (via Element.click()) at the "div" and let the click handler take care of the rest. This lets you reuse existing code rather than duplicating or refactoring.
